wagtail/wagtailadmin/templates/wagtailadmin/notifications/base.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title></title>
+<style>
+{% block styles_content %}
+body {
+	min-width:100%;
+	-webkit-text-size-adjust: none;
+	padding: 0;
+	margin: 0;
+}
+.ReadMsgBody {
+	width: 100%;
+}
+.ExternalClass {
+	width: 100%;
+}
+table {
+	border-collapse:collapse;
+	mso-table-lspace:0pt;
+	mso-table-rspace:0pt;
+}
+table td {
+	border-collapse:collapse;
+}
+td.date a:link {
+	color:#6b6b6b;
+	text-decoration:none;
+}
+@media only screen and (max-width: 568px) {
+body[yahoo] .mobile {
+	width:320px !important;
+}
+body[yahoo] .spacer {
+	width:10px !important;
+}
+body[yahoo] .spacer_height {
+	height:10px !important;
+}
+body[yahoo] .content {
+	width:300px !important;
+}
+body[yahoo] .col-3{
+  float: none !important;
+	width: 100% !important;
+}
+body[yahoo] .force-col {
+	display: block;
+	padding-right: 0 !important;
+	padding-left: 0 !important;
+	padding-bottom: 12px !important;
+}
+body[yahoo] .force-col-last {
+	display: block;
+	padding-right: 0 !important;
+	padding-bottom: 0 !important;
+}
+body[yahoo] .center {
+	text-align:center;
+}
+body[yahoo] .show {
+	display: block !important;
+}
+body[yahoo] .hide {
+	display: none !important;
+}
+body[yahoo] .img {
+	width: 130px !important;
+	float:left;
+}
+body[yahoo] .img img {
+	width:100% !important;
+}
+body[yahoo] .text {
+	width: 150px !important;
+	float:right;
+}
+{% endblock %}
+</style>
+</head>
+{% load i18n %}
+{% load static %}
```

Comment:
I've moved it to the top and changed it to a one-liner. Personally I don't like my "imports" on one line, but it should work now.